### DISCUSSION TITLE
How does github packages replace jfrog bintray?

### DISCUSSION THREAD

**Discussion by jakobneufeld** [Homebrew/discussions#1754] [Everyday usage] [answered]
I am just interested how it works.I looked at the github_packages.rb, I did not understand it, so could you explain it to me. Thanks

**Comment by SMillerDev** [comment, marked as answer]
Github packages allows us to upload tarballs the same as Bintray used to.

**Reply by jakobneufeld** [reply]
But does not github pkgs only support OCI format?

**Reply by SMillerDev** [reply]
https://dlorenc.medium.com/oci-artifacts-explained-8f4a77945c13

**Reply by jakobneufeld** [reply]
Thanks!
